sessionagent/response.go

```diff
@@ -0,0 +1,168 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
```

Comment:
this is all new, as I said, probably a good idea to already place all of this under "usersession/agent"